dds/DCPS/Serializer.cpp

```diff
@@ -196,6 +196,13 @@ bool EncapsulationHeader::to_encoding(
   return true;
 }
 
+void EncapsulationHeader::set_padding_marker(char& options, size_t size)
+{
+  size_t aligned = size;
+  align(aligned, 4);
+  options |= ((aligned - size) & 0x03);
```

Comment:
I don't think we need to call `align()` here, since the running total is never needed.  How about `CONSTANT_FOR_4 - size % CONSTANT_FOR_4`?